src/compositemark/boxplot.ts

```diff
@@ -332,11 +359,13 @@ function boxParams(spec: GenericUnitSpec<Encoding<string>, BoxPlot | BoxPlotDef>
     }
   });
 
+  const window: AggregatedFieldDef[] = aggregate;
+
   return {
     transform: [].concat(
       bins,
       timeUnits,
-      [{aggregate, groupby}],
+      [isMinMax ? {aggregate, groupby} : {window, groupby, frame}],
```

Comment:
You can't just change aggregate to window as the core (non-outliers) layer will render to many data points instead.